src/makeRobotAccountant.js

```diff
@@ -6,7 +6,17 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let counter = 1;
+
+  return (num1) => {
+    const messageOrNum = counter > 3 && counter % 2 === 0
```

Comment:
Create const value for this `counter > 3 && counter % 2 === 0` for better readability 
Or even 2 variables